src/Sylius/Component/Report/Model/ReportInterface.php

```diff
@@ -13,17 +13,68 @@
 
 interface ReportInterface
 {
+    /**
+     * @return mixed
+     */
     public function getId();
+
+    /**
+     * @return string
+     */
     public function getName();
+
+    /**
+     * @param string $name
+     */
     public function setName($name);
+
+    /**
+     * @return string
+     */
     public function getDescription();
+
+    /**
+     * @param string $description
+     */
     public function setDescription($description);
+
+    /**
+     * @return string
+     */
     public function getRenderer();
+
+    /**
+     * @param string $renderer
+     */
     public function setRenderer($renderer);
+
+    /**
+     * @return array
+     */
     public function getRendererConfiguration();
+
+    /**
+     * @param array $rendererConfiguration
+     */
     public function setRendererConfiguration($rendererConfiguration);
```

Comment:
It would be nice to have `array` typehint here: `setRendererConfiguration(array $rendererConfiguration)`